.github/workflows/test.yml

```diff
@@ -7,19 +7,19 @@ jobs:
     name: E2E Tests
     runs-on: ubuntu-latest
     env:
-      POSTGRES_USER: "root"
-      POSTGRES_PASSWORD: "supercoolpassword"
+      POSTGRES_USER: "postgres"
```

Comment:
Using this value and running the test locally, I got this error on Postgres container:
```
2024-02-23 08:38:26 2024-02-23 11:38:26.877 UTC [61] FATAL:  password authentication failed for user "root"
2024-02-23 08:38:26 2024-02-23 11:38:26.877 UTC [61] DETAIL:  Role "root" does not exist.
2024-02-23 08:38:26     Connection matched pg_hba.conf line 99: "host all all all md5"
```

Reply:
> Role "root" does not exist.

@lorerod When you try `postgres` user does it work? As I removed the `root`, and added `postgres` instead.

Reply:
Yes with root it works

Reply:
Can you paste your `.env` here, please?

Reply:
```
# project wide: optional
COMPOSE_PROJECT_NAME=pipeline

# postgrest and postgresql: required environment variables for 'gamma', prod and 'local'
POSTGRES_USER=postgres
POSTGRES_PASSWORD=postgres
POSTGRES_DB=data
POSTGRES_TABLE=medic # for dbt use only
POSTGRES_SCHEMA=v1

# dbt: required environment variables for 'gamma', 'prod' and 'local'
DBT_POSTGRES_USER=postgres
DBT_POSTGRES_PASSWORD=postgres
DBT_POSTGRES_SCHEMA=dbt
DBT_POSTGRES_HOST=postgres # Your postgres instance IP or endpoint in "prod".
CHT_PIPELINE_BRANCH_URL="https://github.com/medic/cht-pipeline.git#hardcoded-variables"

# couchdb and logstash: required environment variables for 'gamma', 'prod' and 'local'
COUCHDB_USER=medic
COUCHDB_PASSWORD=password
COUCHDB_DBS="medic" # space separated list of databases you want to sync e.g "medic medic_sentinel"
COUCHDB_HOST=couchdb
COUCHDB_PORT=5984
COUCHDB_SECURE=false

# superset: required environment variables for 'gamma', 'prod' and 'local'
SUPERSET_PASSWORD=password
SUPERSET_ADMIN_EMAIL=user@superset.com
SUPERSET_SSH_TUNNELING=True
```

Reply:
You are right. I download all the images again, including the new `medicmobile/dataemon`; it is now working using postgres as user.
You can ignore my suggestion.

Reply:
Perfect, thanks for confirming Lorena! I will update the branch to `main`, and send the PR as ready for review.